src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -834,12 +870,11 @@ private void CoinstakeWorker(CoinstakeWorkerContext context, ChainedBlock chainT
                     context.Logger.LogTrace("Trying with transaction time {0}...", txTime);
                     try
                     {
-                        var prevoutStake = new OutPoint(utxoStakeInfo.UtxoSet.TransactionId, utxoStakeInfo.OutputIndex);
-                        long nBlockTime = 0;
+                        var prevoutStake = new OutPoint(utxoStakeInfo.UtxoSet.TransactionId, utxoStakeInfo.OutPoint.N);
 
                         var contextInformation = new ContextInformation(new BlockValidationContext { Block = block }, this.network.Consensus);
                         contextInformation.SetStake();
-                        this.posConsensusValidator.StakeValidator.CheckKernel(contextInformation, chainTip, block.Header.Bits, txTime, prevoutStake, ref nBlockTime);
+                        this.posConsensusValidator.StakeValidator.CheckKernel(contextInformation.Stake, chainTip, block.Header.Bits, txTime, prevoutStake, out _);
```

Comment:
pls don't use `_`, use `unused` instead or similar
but here, probably just remove the out parameter from the codebase entirely, it is not used anywhere